posix.mak

```diff
@@ -312,6 +312,13 @@ test/%/.run: test/%/Makefile $(DMD)
 		DRUNTIME=$(abspath $(DRUNTIME)) DRUNTIMESO=$(abspath $(DRUNTIMESO)) LINKDL=$(LINKDL) \
 		QUIET=$(QUIET) TIMELIMIT='$(TIMELIMIT)' PIC=$(PIC)
 
+################### tests ######################################
```

Comment:
The existing posix tests are listed in variable ADDITIONAL_TESTS and expect just a Makefile in the test folder.

Reply:
Oh right. Wow.
This setup is ridiculous, that's no consistency at all. This is **sooooo** time consuming >_<

Reply:
Agreed. The legacy is using underpowered DigitalMars make on Windows, though. I think there shouldn't be two ways to run tests in posix.mak.